lib/active_record/connection_adapters/cockroachdb_adapter.rb

```diff
@@ -247,12 +247,9 @@ def initialize_type_map(m = type_map)
             precision = extract_precision(sql_type)
             scale = extract_scale(sql_type)
 
-            # TODO(#178) this should never use DecimalWithoutScale since scale
-            # is assumed to be 0 if it is not explicitly defined.
-            #
```

Comment:
I think this comment should still be here since ideally we wouldn't need this conditional and `OID::Decimal` would properly handle the case where precision is defined, but the scale is not.

Reply:
Yeah, I removed it because it's not in the original rails code and the new logic is much closer to the original logic in rails.

Reply:
Yes, you're right. I think issue #178 can be closed since this handles the case described in that issue (`DECIMAL` where only precision is specified).

Reply:
Actually, can you add this back in now? I'm going to add to that issue to reflect the problem we saw with precision being improperly handled.

Reply:
I verified it does the same behavior in vanilla activerecord.